common/src/main/java/net/whimxiqal/journey/data/versionmanager/DataVersionManager.java

```diff
@@ -21,16 +21,18 @@
  * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-package net.whimxiqal.journey.data;
+package net.whimxiqal.journey.data.versionmanager;
 
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.FileWriter;
 import java.io.IOException;
 import java.util.Scanner;
 import net.whimxiqal.journey.Journey;
+import net.whimxiqal.journey.data.DataVersion;
 
-public class DataVersionManager {
+public interface DataVersionManager {
```

Comment:
This works great, but just to align it with the style of the existing code, "Managers" are usually stored for a long period of time and are used to track and manage internal state through an interface. This one is confusing because this class only retrieves the version and doesn't do anything else to manage it.

I would prefer this class to merge with `MigrationHandler` and use the name `DataVersionHandler`. It makes sense to me that the handler of the `DataVersion` would be able to:
1. retrieve the stored `DataVersion`
2. store a new `DataVersion`
3. run migrations to update the data version